source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+* ``DEC`` tform: ``1E``, unit: deg
+    * Reconstructed event Declination (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``ENERGY`` tform: ``1E``, unit: TeV
+    * Reconstructed event energy.
+      See also the `OGIP event list`_ standard.
+* ``DETX`` tform: ``1E``, unit: deg
+    * Reconstructed event X-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``DETY`` tform: ``1E``, unit: deg
+    * Reconstructed event Y-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``EVENT_TYPE`` tform: ``32X``
+    * Event quality partition.
+
+      .. note::
+         It needs to be discussed whether this should be a mandatory column
+         or an optional column. The partition of events into quality classes
+         seems sufficiently fundamental so that all IACT should in principle
+         support this. For example event multiplicity could be translated
+         into quality partitions, and the IRFs computed accordingly.
+
+
+Optional columns
+----------------
+
+.. note::
+   None of the following columns is required to be part of an ``EVENTS``
+   extension. Any software using these columns should first check whether the
+   columns exist, and warn in case of their absence.
+
+* ``ALT`` tform: ``1E``, unit: deg
+    * Reconstructed altitude coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``AZ`` tform: ``1E``, unit: deg
+    * Reconstructed azimuth coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``THETA`` tform: ``1E``, unit: deg
+    * Reconstructed offset from the observation pointing position
+* ``PHI`` tform: ``1E``, unit: deg
+    * Reconstructed position angle from the observation pointing position
+      (position angles are counted counterclockwise from celestial North)
+* ``MULTIP`` tform: ``1I``
+    * Telescope multiplicity. Number of telescopes that have seen the event
+* ``DIR_ERR`` tform: ``1E``, unit: deg
+    * Direction error of reconstruction 
+* ``ENERGY_ERR`` tform: ``1E``, unit: TeV
+    * Error on reconstructed event energy
+* ``COREX`` tform: ``1E``, unit: m
+    * Core position X of shower
+* ``COREY`` tform: ``1E``, unit: m
+    * Core position Y of shower
+* ``CORE_ERR`` tform: ``1E``, unit: m
+    * Error on core position of shower    
+* ``XMAX`` tform: ``1E``, unit: radiation lengths
+    * First interaction depth 
+* ``XMAX_ERR`` tform: ``1E``, unit: radiation lengths
+    * Error on first interaction depth 
+* ``HIL_MSW`` tform: ``1E``
+    * Hillas mean scaled width
+* ``HIL_MSW_ERR`` tform: ``1E``
+    * Hillas mean scaled width error
+* ``HIL_MSL`` tform: ``1E``
+    * Hillas mean scaled length
+* ``HIL_MSL_ERR`` tform: ``1E``
+    * Hillas mean scaled length error
+
+
+Mandatory header keywords
+-------------------------
```

Comment:
It's indeed a significant change, but formally it's just moving information from a header into a binary table, and we know anyway that we want to go this way. I agree that it requires adaption of some code, but this should be rather minor. As we are working on v0 now, we formally don't have to worry for backward-compatibility, we now define the reference from which we will start. So I think in one or two months from now, it will decrease our headaches, and we will gain tremendously in flexibility (we don't need to use it though for a long time).

I agree that `DEADC` could also be moved to into the pointing table (it should not go into `GTI` as these may be changed by the user). You don't need to do that for `LIVETIME`, you even do not need `LIVETIME` as it can be inferred from `DEADC` and `ONTIME`.

However, these quantities still make sense as average values for the entire event file, fulfilling thus the OGIP standard. I would therefore always keep this information in the header, at least for user information, even if a more fine grained information is used later for TECH3.